Makefile

```diff
@@ -29,6 +29,13 @@ test:
 	docker-compose up -d db queue
 	docker-compose -f docker-compose.test.yml up --build --force-recreate --exit-code-from unit_test unit_test
 
+newman:
+	# This target should be executed by passing in an argument for the environment (dev/test/sbx)
+	# and if needed a token.
+	# For example: make newman env=test token=<MY_TOKEN>
+	docker build . -f Dockerfiles/Dockerfile.newman_test -t newman_test
```

Comment:
Since this is intended to execute against a running application, you'll need to bring up the app similar to the other test targets.  Additionally, the test container should really be tied into the `docker-compose.test.yml` file so all the networking just "works".